pom.xml

```diff
@@ -57,6 +57,22 @@
 			<version>18.0</version>
 		</dependency>
 		<dependency>
+			<groupId>org.hamcrest</groupId>
+			<artifactId>hamcrest-core</artifactId>
+			<version>1.3</version>
+		</dependency>
+		<dependency>
+			<groupId>net.java.dev.jna</groupId>
+			<artifactId>jna</artifactId>
+			<version>4.1.0</version>
```

Comment:
What do you think to add <scope>test</scope> to the tweo net.java.dev.jna dependencies? I think we need this dependency only for integration tests?